index.js

```diff
@@ -0,0 +1,102 @@
+import './modules/navigation.js';
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Validating form
+
+// ====== Handle local Storage =====
+
+const Store = {
+  getBooks() {
+    let bookInputs;
+    if (localStorage.getItem('bookInputs') === null) {
+      bookInputs = [];
+    } else {
+      bookInputs = JSON.parse(localStorage.getItem('bookInputs'));
+    }
+    return bookInputs;
+  },
+
+  addBook(book) {
+    const bookInputs = Store.getBooks();
+    bookInputs.push(book);
+    localStorage.setItem('bookInputs', JSON.stringify(bookInputs));
+  },
+
+  removeBook(del) {
+    const bookInputs = Store.getBooks();
+    bookInputs.forEach((book, index) => {
+      if (book.del === del) {
+        bookInputs.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('bookInputs', JSON.stringify(bookInputs));
+  },
+};
+
+// ==== Handle UI ====
+
+const page = {
+  displayBooks() {
+    const newBooks = Store.getBooks();
+    newBooks.forEach((books) => page.bookList(books));
+  },
+
+  bookList(book) {
+    const list = document.getElementById('book-collection');
+    const create = document.createElement('div');
+    create.className = 'list-control';
+
+    const p = document.createElement('p');
+    const b = document.createElement('b');
+    b.textContent = ' By ';
+    p.append(book.title, b, book.author);
+
+    const btn = document.createElement('button');
+    btn.classList.add('remove-btn');
+    btn.textContent = 'Remove';
+
+    create.append(p, btn);
+    list.appendChild(create);
+  },
```

Comment:
- [ ] I would suggest that you continue to break up your codes located here in `index.js` file so that you exercise more with modular structuring 😃 so please make sure **that you group some of the functions here in this file into its separate file in the modules folder** 😉